presets/presets.lua

```diff
@@ -1077,74 +1077,189 @@ presets = {
+            {type="technology", name="fi_stage_tech", quantity=1},
+            {type="technology", name="fi_energy_tech", quantity=1},
+            {type="technology", name="fu_stage_tech", quantity=1},
+            {type="technology", name="fu_energy_tech", quantity=1},
+            {type="technology", name="gr_stage_tech", quantity=1},
+
+            {type="group",      name="vanilla"},
+            {type="fluid",      name="petroleum-gas", quantity=1},
+            {type="item",       name="locomotive", quantity=1},
+            {type="item",       name="construction-robot", quantity=1},
+            {type="item",       name="logistic-robot", quantity=1},
+            {type="group",      name="Electronics"},
+            {type="item",       name="el_materials_ALK", quantity=1},
+            {type="item",       name="el_energy_crystal_item", quantity=1},
+            {type="item",       name="el_diesel_train_item", quantity=1},
+
+            {type="group",      name="Fisson"},
+            {type="item",       name="fi_materials_glass", quantity=1},
+            {type="item",       name="fi_materials_gold", quantity=1},
+            {type="item",       name="fi_materials_neodym", quantity=1},
+            {type="item",       name="fi_materials_titan", quantity=1},
+            {type="item",       name="fi_materials_GFK", quantity=1},
+            {type="item",       name="fi_materials_NFK", quantity=1},
+            {type="item",       name="fi_energy_crystal_item", quantity=1},
+            {type="item",       name="fi_solid_reactor_item", quantity=1},
+
+            {type="group",      name="Fusion"},
+            {type="item",       name="fu_materials_KFK", quantity=1},
+            {type="item",       name="fu_materials_TIM", quantity=1},
+            {type="item",       name="fu_materials_fusion_card", quantity=1},
+            {type="item",       name="fu_materials_plasma_card", quantity=1},
             {type="item",       name="fu_materials_laser_card", quantity=1},
-            {type="item",       name="fu_materials_magnet_card",quantity=1},
-            {type="item",       name="fu_tech_sign_item",       quantity=1},
-            {type="item",       name="fu_stelar_reactor_item",  quantity=1},
+            {type="item",       name="fu_materials_magnet_card", quantity=1},
+            {type="item",       name="fu_tech_sign_item", quantity=1},
+            {type="item",       name="fu_robo_logistic_item", quantity=1},
+            {type="item",       name="fu_robo_construction_item", quantity=1},
+            {type="item",       name="fu_stelar_reactor_item", quantity=1},
             {type="item",       name="fu_tokamak_reactor_item", quantity=1},
-            {type="item",       name="fu_star_engine_core_item",quantity=1},
+            {type="item",       name="fu_star_engine_core_item", quantity=1},
+
+            {type="group",      name="Exotics"},
             {type="item",       name="fu_space_probe_science_item", quantity=1},
```

Comment:
Is that one meant to be in "Fusion"?

Reply:
no as althought its unlocked at the end of the fusion tier its used throughout the exotics tier